configurations/kms-ear.yaml

```diff
@@ -0,0 +1,11 @@
+
+scylla_encryption_options: "{ 'cipher_algorithm' : 'AES/ECB/PKCS5Padding', 'secret_key_strength' : 128, 'key_provider': 'KmsKeyProviderFactory', 'kms_host': 'kms_test'}"
```

Comment:
Shouldn't it be `KmsRealKeyProviderFactory` instead of the `KmsKeyProviderFactory`?